curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -116,6 +131,8 @@ void FuseOpInit(void *userdata, struct fuse_conn_info *conn) {
     if (ret != CURVEFS_ERROR::OK) {
         LOG(FATAL) << "FuseOpInit failed, ret = " << ret;
     }
+
+    EnableSplice(conn);
```

Comment:
EnableSplice can be set in g_ClientInstance->FuseOpInit()

Reply:
it ok to put it in here